source/extensions/omni.isaac.lab/docs/CHANGELOG.rst

```diff
@@ -1,6 +1,25 @@
 Changelog
 ---------
 
+0.26.1 (2024-10-14)
```

Comment:
Major bump since we break stuff....

Reply:
Can we make it somewhat backwards compatible? So throw a warning if someone uses NullCommand class, and we tell them to set command to None instead? Ideally if you are just playing around, you may just set it to NullCommand for testing and then change it to something else once you are ready.

Changing the variable to None seems more effortful.